tools/fiptool/fiptool.c

```diff
@@ -561,10 +570,29 @@ static void parse_plat_toc_flags(char *arg, unsigned long long *toc_flags)
 	*toc_flags |= flags << 32;
 }
 
+static int is_power_of_2(unsigned long x)
+{
+        return x && !(x & (x - 1));
+}
```

Comment:
I did not know we can include include/lib/utils.h from host programs.
You need to be very careful about this
because mixture of target image code and host program code
often leads to maintenance headache.

Actually, we need to share include/common/firmware_image_package.h
between BL1/BL2 and FIPTOOL.  But, this is just one of some exceptions.

Currently, we do not have clear boundary between BL images and host-tools.
I believe this kind of work is really worthwhile.

At this moment, I do not want to include utils.h from FIPTOOL.
If we want to do it, we need to define a "shareable header directory" 
and move utils.h there.  But, this is a separate work.

For this pull request, I am opposed to including <utils.h> only for borrowing IS_POWER_OF_TWO().